dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/TaskType.java

```diff
@@ -58,7 +59,8 @@
     SEATUNNEL(13, "SEATUNNEL"),
     SWITCH(14, "SWITCH"),
     PIGEON(15, "PIGEON"),
-    DATA_QUALITY(16, "DATA_QUALITY");
+    DATA_QUALITY(16, "DATA_QUALITY"),
+    EMR(17, "EMR");
```

Comment:
A small tips, add ',' at the enum end can help to reduce changes when someone a new enum in the future.
```suggestion
    DATA_QUALITY(16, "DATA_QUALITY"),
    EMR(17, "EMR"),
    ;
```